app/forms/providers/means/regular_income_form.rb

```diff
@@ -66,6 +66,10 @@ def none_selected
         "none"
       end
 
+      def frequencies(transaction_type)
+        transaction_type.name == "benefits" ? RegularTransaction::FREQUENCIES.without("monthly") : RegularTransaction::FREQUENCIES
+      end
```

Comment:
How do you feel about adding a class method on the `RegularTransaction` model instead and adding some unit tests?

Maybe something like this:

```ruby
class RegularTransaction
  def self.frequencies_for(transaction_type)
    if transaction_type.name == "benefits"
      FREQUENCIES.without("monthly")
    else
      FREQUENCIES
    end
  end
end
```

I feel like the context of regular transaction frequencies is broader than this form specifically, so this might help capture that. 

What do you think?

Reply:
Yeah that makes sense, I've updated it to basically move this method to the RegularTransaction model